src/unix/pthread.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+
+#ifndef GIT_PTHREAD_H
+#define GIT_PTHREAD_H
```

Comment:
Heh, I've seen it being used like this in `win32/pthread.h` and just re-used it, assuming it was being done like this for internal headers while externally visible headers used the `INCLUDE_` prefix. Will change.